lib/hammer_cli/output/definition.rb

```diff
@@ -12,10 +12,117 @@ def append(fields)
       @fields += fields
     end
 
+    def delete_fields!(by = {})
+      return clear if by.empty?
+      return deep_delete!(@fields, by) if by[:deep_delete]
+      ids, labels = adjust_options(by)
+      @fields.delete_if do |f|
+        label = f.label
+        label = f.label.downcase unless label.nil?
+        ids.include?(f.id) || labels.include?(label)
+      end
+    end
+
+    def select_fields!(by = {})
+      return self if by.empty?
+      return @fields = deep_select(@fields, by) if by[:deep_select]
+      ids, labels = adjust_options(by)
+      @fields.keep_if do |f|
+        label = f.label
+        label = f.label.downcase unless label.nil?
+        ids.include?(f.id) || labels.include?(label)
+      end
+    end
+
+    def replace_field(field_id, fields = nil, &block)
+      i = @fields.find_index do |f|
+        f.id == field_id
+      end
+      return if i.nil?
+      definition = build_definition(fields, &block)
+      multiple_replace(i, definition.fields)
+    end
+
+    def at(path = [])
```

Comment:
I like that method. Could the path also accept a single symbol? Eg `at(:interfaces)`